<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" parallel="tests" name="Error validation(failed)">
    <classes>
      <class name="rabinaqueendom.Tests.ErrorValidationTest">
        <methods>
          <include name="submitOrders"/>
          <include name="launchApplication"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- rabinaqueendom.Tests.ErrorValidationTest -->
    </classes>
  </test> <!-- Error validation(failed) -->
  <test thread-count="5" parallel="tests" name="Submit Order Test(failed)">
    <classes>
      <class name="rabinaqueendom.Tests.StandAloneTest">
        <methods>
          <include name="launchApplication"/>
          <include name="tearDown"/>
          <include name="submitOrders"/>
          <include name="orderList"/>
        </methods>
      </class> <!-- rabinaqueendom.Tests.StandAloneTest -->
    </classes>
  </test> <!-- Submit Order Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
